src/time_tracker/invoices/handlers/spec.clj

```diff
@@ -45,6 +45,5 @@
 (s/def ::timers (timers-spec/normalized-timers-spec 0))
 
 (s/def ::invoice-data
-  (s/and (s/merge ::generate-invoice-params-coerced
-                  (s/keys :req-un [::users-spec/users ::timers-spec/timers]))
-         invoice-data-pred))
```

Comment:
@jysandy The predicate matches 2 sets of users. One that it builds from the timers and another that it builds from the submitted user-rates. The first one is the set of all Users since they come from screen and the screen has all Users even if a few of them are staffed on that project. The second set of Users are now restricted to the the ones with Timers just on the project. So they do not match anymore